app/Services/Commands/Commands/DnsLookup.php

```diff
@@ -18,9 +18,13 @@ public function perform(string $command): Response
     {
         $dns = new Dns($command);
 
-        $dnsRecords = $dns->getRecords();
+        try {
+            $dnsRecords = $dns->getRecords();
 
-        $domain = $dns->getDomain($command);
+            $domain = $dns->getDomain($command);
+        } catch (\Exception $e) {
```

Comment:
Let's always import namespaces. I'll quickly do this myself.

Reply:
Thanks! For some reason PHPStorm never imports classes with the root namespace. Gonna ask them about this.